%YAML:1.2
---
PlanMap:
  resolution: 0.1
  sogm_size_x: 16
  sogm_size_y: 16
  sogm_size_z: 6
  p_hit: 0.70
  p_miss: 0.35
  p_min: 0.12
  p_max: 0.97
  p_occ: 0.80

  useSquareDist: 0 # 0 = false, 1=true
  esdf_max_xy: 3.0
  esdf_max_z: 3.0


DepthCamera:

  #  heigth: 640 # /camera/depth/image_raw
  #  width: 480
  #  fx: 565.6009
  #  fy: 565.6009
  #  cx: 320.5
  #  cy: 240.5

  # heigth: 640 # /realsense_plugin/camera/depth/image_raw
  # width: 480
  # fx: 554.3826904296875
  # fy: 554.3826904296875
  # cx: 320.0
  # cy: 240.0

  heigth: 640 # /D455/depth/image_rect_raw
  width: 480
  fx: 392.365234375
  fy: 392.365234375
  cx: 320.01055908203125
  cy: 241.12890625

  # heigth: 640 # /pcl_render_node/depth
  # width: 480
  # fx: 387.229248046875
  # fy: 387.229248046875
  # cx: 321.04638671875
  # cy: 243.44969177246094

  k_depth_scaling_factor: 1000
  depth_maxdist: 3.0
  depth_mindist: 0.1
  depth_filter_margin: 0
  skip_pixel: 4

  R_C_2_B: !!opencv-matrix #normal realsense
    rows: 3
    cols: 3
    dt: d
    data: [0, 0, 1, -1, 0, 0, 0, -1, 0]


  T_C_2_B: !!opencv-matrix  # !! explicity specify the type
    rows: 1
    cols: 3
    dt: d
    data: [0.0, 0.0, 0.0]
